src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -795,13 +765,13 @@ public bool Equals(String value, StringComparison comparisonType)
                 case StringComparison.OrdinalIgnoreCase:
                     if (this.Length != value.Length)
                         return false;
-
+#if CORECLR
```

Comment:
Why do we need these ifdefs now?

Reply:
These ifdefs will be needed once this file is shared with CoreRT. I have added them to verify that the change will work well for CoreRT.